src/lib/chat/matrix-client.ts

```diff
@@ -457,6 +472,15 @@ export class MatrixClient implements IChatClient {
     return '';
   }
 
+  private getRoomAvatar(room: Room): string {
```

Comment:
We could add a helper function for all of these 'get' methods at some point 🙂 will take a look if I happen to work around these